setup.cfg

```
@@ -59,6 +60,7 @@ envlist =
     py{38,39,310,311}-dj{42,50}-redislatest
```

you have to add python312 here too, I've also added django 5.1 since I've added it in #754 
```suggestion
    py{38,39,310,311,312}-dj{42,50,51}-redislatest
```